### DISCUSSION TITLE
server: ability for watchers to put comments

### DISCUSSION THREAD

**Discussion by daixtr** [asciinema/discussions#207] [Ideas]
It will appear under the player, such that those watching the show can put comments about it and thus collaborate or make corrections / enhancement of what the show is all about...

**Comment by ku1ik** [comment]
We used to have commenting few years ago but it wasn't very much used. It was removed during the redesign. I'll consider adding it back if people are interested.

**Comment by unixfox** [comment]
👍 Comments are always good for watchers to react about the asciinema video.

**Comment by Tau5** [comment]
I am interested in this addition

**Comment by ku1ik** [comment]
Let's do this 👍
